modules/serverless-nodejs-context-object-reference.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-functions-reference-guide.adoc
+
+[id="serverless-nodejs-context-object-reference_{context}"]
+= Node.js context object reference
+
+The `context` object has several properties that may be accessed by the function developer.
+
+[id="serverless-nodejs-context-object-reference-log_{context}"]
+== log
+
+Provides a logging object that can be used to write output to the cluster logs. The log adheres to the Pino logging API (https://getpino.io/#/docs/api).
+
+.Example
+[source,js]
+----
+Function myFunction(context) {
```

Comment:
```suggestion
function handle(context) {
```
I think, similar to the "return values" adoc, all of our examples should just be `function handle()`. Note the lower case `f` in `function`. Sorry for the last minute recommendation here.